package.json

```diff
@@ -2,26 +2,15 @@
   "name": "cozy-desktop",
   "version": "0.7.0",
   "description": "File Synchronization Client for Cozy Cloud",
-  "main": "app.html",
-  "window": {
-    "position": "center",
-    "width": 600,
-    "height": 600,
-    "show": true,
-    "resizable": false,
-    "toolbar": false,
-    "icon": "client/public/icon/main_icon.png",
-    "title": "Cozy Desktop"
-  },
-  "scripts": {
-    "lint": "gulp lint",
-    "test": "gulp test",
-    "build": "gulp build",
-    "prepublish": "gulp build",
-    "start": "nw"
+  "homepage": "https://github.com/cozy-labs/cozy-desktop",
+  "author": "Cozy Cloud <contact@cozycloud.cc> (https://cozycloud.cc/)",
+  "license": "LGPL-3.0",
+  "bugs": {
+    "url": "https://github.com/cozy-labs/cozy-desktop/issues"
   },
+  "main": "./lib/bin/cli.js",
```

Comment:
I copy from this package:
- https://github.com/clutchski/coffeelint/blob/master/package.json#L12
- https://github.com/mauricemach/coffeekup/blob/master/package.json#L12

but I have not opinion for this ;)